subcommand/server-acl-init/rules.go

```diff
@@ -212,16 +212,18 @@ func (c *Command) injectRules() (string, error) {
 operator = "write"
 {{- end }}
 {{- if .EnableHealthChecks }}
+node_prefix "" {
+  policy = "write"
+}
 {{- if .EnableNamespaces }}
 namespace_prefix "" {
-{{- end }}
-  node_prefix "" {
-     policy = "write"
-  }
   service_prefix "" {
-     policy = "write"
+    policy = "write"
   }
-{{- if .EnableNamespaces }}
+}
+{{- else }}
+service_prefix "" {
+  policy = "write"
 }
 {{- end }}
```

Comment:
Something like:
```
{{- if .EnableHealthChecks }}
node_prefix "" {
  policy = "write"
}
{{- if .EnableNamespaces }}
namespace_prefix "" {
{{- end }}
  service_prefix "" {
    policy = "write"
  }
{{- if .EnableNamespaces }}
}
{{- end }}
```
It seems to have been our approach in other places in this file but I could be talked out of it

Reply:
I had initially thought this was a good route, but then we end up with spacing problems on the service_prefix portion, which wouldn't be indented if namespaces were disabled.
I guess the tradeoff is ugly resultant hcl vs slightly more complex generation code.
Thoughts?

Reply:
Yeah this bothers me too 😂 . But let's just follow the existing pattern for now and then maybe we can change it for all of them at some point.

Reply:
Yeah. Im with Luke. Let's be consistent now and we can re-jigger the entire file down the road.

Reply:
alrighty, can do!